services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -63,46 +72,48 @@ async def get_active_project(request: web.Request) -> web.Response:
 
 
 #
-# Custom methods https://google.aip.dev/136
+# open project: custom methods https://google.aip.dev/136
 #
 
 
-@routes.post(f"/{VTAG}/projects/{{project_uuid}}:open")
+@routes.post(f"/{VTAG}/projects/{{project_uuid}}:open", name="open_project")
 @login_required
 @permission_required("project.open")
 async def open_project(request: web.Request) -> web.Response:
-    user_id: int = request[RQT_USERID_KEY]
+    c = BaseRequestContext.parse_obj(request)
```

Comment:
in the description of your PR you have a note that the context validation is not always necessary. can you maybe be more precise? is that only to get the user_id?

Reply:
There is the app context and the request context. The variables stored in each context should be validated when it gets initialized. Therefore in a handler it does not make sense to validate everytime the app variables. That was my point.